docs/tools/debugging.md

```diff
@@ -55,3 +59,47 @@ given destination directory. Each archive will contain:
 
 Note: goroutine.out and heap.out will only be written if a profile address is
 provided and is operational. This command is blocking and will log any error.
+
+## CometBFT Inspect
```

Comment:
The `inspect` command is not present on v0.34, this section should be removed.

Reply:
removed the inspect command information, thanks.